### DISCUSSION TITLE
Changed array data types

### DISCUSSION THREAD

**Discussion by Fudal** [hasura/graphql-engine#9843] [General] [answered]
After upgrade hasura engine version from 2.19.0 to 2.32.0 I have that error:
`GraphQLError(message: variable 'repeated_weekdays' is declared as '_int4', but used where '[Int!]' is expected)`

in my Flutter code I have this:
`\$repeated_weekdays: _int4`

I needed to downgrade hasura engine to 2.19.0 for remove that error.
How can I upgrade my hasura engine to newer version without this error and without update my Flutter app in the same time?

It is not possible to update the database and the mobile app at the same time, because not all users of the app will update it immediately, so for those users with an old version, the database will throw an error.

**Comment by SamirTalwar** [comment, marked as answer]
You can opt into the `disable_postgres_arrays` experimental feature flag (using the `--experimental-features` option, or the `HASURA_GRAPHQL_EXPERIMENTAL_FEATURES` environment variable).
